injector/network_disruption.go

```diff
@@ -459,13 +459,13 @@ func (i *networkDisruptionInjector) addServiceFilters(serviceName string, filter
 	builtServices := []tcServiceFilter{}
 
 	for _, filter := range filters {
-		i.config.Log.Infow("found service endpoint", "resolvedEndpoint", filter.service.String(), "resolvedService", serviceName)
-
 		filter.priority, err = i.config.TrafficController.AddFilter(interfaces, "1:0", "", nil, filter.service.ip, 0, filter.service.port, filter.service.protocol, network.ConnStateUndefined, flowid)
 		if err != nil {
 			return nil, err
 		}
 
+		i.config.Log.Infow(fmt.Sprintf("added a tc filter for service %s-%s with priority %d", serviceName, filter.service.String(), filter.priority), "interfaces", strings.Join(interfaces, ", "))
```

Comment:
```suggestion
		i.config.Log.Infow(fmt.Sprintf("added a tc filter for service %s-%s with priority %d", serviceName, filter.service, filter.priority), "interfaces", interfaces)
```
How does it look? It should be pretty similar IMO